census/src/main/scala/grasshopper/census/search/AddressInterpolator.scala

```diff
@@ -1,30 +1,16 @@
 package grasshopper.census.search
 
-import geometry._
 import feature._
-import io.geojson.FeatureJsonProtocol._
+import geometry._
 import grasshopper.census.model.AddressRange
 
 object AddressInterpolator {
 
   def calculateAddressRange(f: Feature, a: Int): AddressRange = {
```

Comment:
Can we use more descriptive parameter names, especially for public functions.  We're using `feature` in `interpolate()`, and I think `a` is the `addressNumber`?